hooks/afterRegistration.ts

```diff
@@ -1,5 +1,5 @@
 // This function will be fired both on server and client side context after registering other parts of the module
-export function afterRegistration(Vue, config, store, isServer){
```

Comment:
ok with `config`, but dont understand, why `isServer` removed too?

Reply:
@Devors it was used from `Vue.prototype` but I switched to using from argument now 👍